docker/compose-extras.yml.sample

```diff
@@ -1,5 +1,3 @@
 version: '3.3'
 #services:
 #  wordpress:
-#    volumes:
```

Comment:
Do you think we should add a comment here pointing to the new solution instead?

Reply:
Sure, we can do that.